honeybee_3dm/face.py

```diff
@@ -0,0 +1,145 @@
+import rhino3dm
```

Comment:
We need a couple of lines here to explain what this module is supposed to do.

Reply:
Complied.